### PR TITLE
Feature/issue 1236 primary key null on clone

### LINKED ISSUE #1236: FEAT: Set primary key cell to NULL while cloning a row
Currently while cloning a row, one of the things that needs to be done for primary keys (especially when they are auto incremented) is to delete the current value and keep it as NULL before applying your changes. 

It would be great if when cloning a row the primary key cell would be set to NULL auomatically on the new cloned row, at least when the primary key has auto increment setting.

Now that multiple rows selection is around the corner, it feels it would have even more need.

Thank you

### PR DESCRIPTION
**Pull request by wmontgomery**
Fix #1236

### CODE REVIEW

File: apps/studio/src/components/tableview/TableTable.vue [hunk 1]
```diff
@@ -927,8 +927,14 @@ export default Vue.extend({
     cellCloneRow(_e, cell) {
       const row = cell.getRow()
       const data = { ...row.getData() }
+      const dataParsed = Object.keys(data).reduce((acc, d) => {
+        if (!this.primaryKeys.find(element => element === d)) {
```
Comment (rathboma) [inline apps/studio/src/components/tableview/TableTable.vue:931]: This should just use `includes` -> `if(!this.primaryKeys?.includes(d))`

**Review by rathboma — CHANGES_REQUESTED** [review]
Find => includes, then we're good!

File: apps/studio/src/components/tableview/TableTable.vue (hunk 1 above)
Comment (wmontgomery) [reply apps/studio/src/components/tableview/TableTable.vue]: Includes it is!

File: apps/studio/src/store/index.ts [hunk 2]
```diff
@@ -424,6 +423,9 @@ const store = new Vuex.Store<State>({
             })
           })
 
+          // eslint-disable-next-line no-debugger
+          // debugger
```
Comment (rathboma) [inline apps/studio/src/store/index.ts:427]: Three leave, one enters :laughing:

**Pull request merged**